<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BSTsence</class>
 <widget class="QMainWindow" name="BSTsence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addNode">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>添加结点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NodeInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeIT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actioncreatRandomTree">
   <property name="text">
    <string>creatRandomTree</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
